docs/respondent_management_to_electronic_questionnaire.rst

```diff
@@ -25,6 +25,12 @@ Schema Definition
     The trading as name for a responding unit. Temporary until wider refactor.
   eq_id
     The eQ questionnaire instance id.
+  case_id
+    The case UUID used to identify a single instance of a survey collection for a respondent
+  case_ref
+    The case reference identified by the above UUID (e.g. "1000000000000001")
```

Comment:
`case_id`, `case_ref` and `account_service_url` should be documented as optional